src/uu/tail/src/paths.rs

```diff
@@ -89,6 +80,7 @@ impl Input {
     pub fn is_tailable(&self) -> bool {
         match &self.kind {
             InputKind::File(path) => path_is_tailable(path),
+            // TODO: shouldn't the default be true instead of false??
```

Comment:
Why? If we can't find the path, how can we tail it? Or do you want to print an error while tailing?

Reply:
This method together with `Input::resolve` was removed from `Input` in the main refactoring pr, so this TODO can be ignored and I'm removing it. Sorry, I just quickly added this TODO at that time because I wanted to think about it, but later, and forgot to remove it from this pr.